bindings/python/pyproject.toml

```
@@ -34,11 +34,15 @@ name = "pyiceberg-core"
 readme = "project-description.md"
 requires-python = "~=3.9"
 dynamic = ["version"]
+license = { file = "LICENSE" }
 
 [tool.maturin]
 features = ["pyo3/extension-module"]
 module-name = "pyiceberg_core.pyiceberg_core_rust"
 python-source = "python"
+include = [
+    { path = "NOTICE", format = ["sdist", "wheel"] }
+]
```

```suggestion
include = [
    { path = "LICENSE", format = ["sdist", "wheel"] }
    { path = "NOTICE", format = ["sdist", "wheel"] }
]
```

should we also include LICENSE here?

---

even if this is a no-op, i would prefer adding this to be explicit.

---

Done, please review again!